spec/models/generic_file_spec.rb

```diff
@@ -519,4 +519,90 @@
       end
     end
   end
+
+  let(:mock_gv_black_collection) { [{title: "mock gv black collection", id: GenericFile::GV_BLACK_COLLECTION_ID}] }
+  let(:mock_gv_black_photo_collection) { [{title: "gv black photograph sub collection", id: GenericFile::GV_BLACK_PHOTOGRAPH_SUB_COLLECTION_ID}] }
+  let(:mock_collection) { [{title: "test collection", id: "test-collection"}] }
+
+  let(:collection_paths_gv_black_not_in_photo) { [mock_gv_black_collection] }
+  let(:collection_paths_gv_black_in_photo) { [mock_gv_black_collection, mock_gv_black_photo_collection] }
+  let(:no_gv_black) { [mock_collection] }
+
+  # if it's in the old gv black collection and not the photograph sub collection
+  describe "#in_old_gv_black_and_not_photograph" do
+    context "in old gv black collection" do
+      context "in photograph sub collection" do
+        it "returns false" do
+          expect(gf_doi.send(:in_old_gv_black_and_not_photograph?, collection_paths_gv_black_in_photo)).to eq(false)
```

Comment:
minor:

1- Although the structure `collection_paths = [ [gv_black], [gv_black_photo] ]` will pass the tests, in reality the structure is `[ [gv_black, gv_black_photo], [other_random_collection] ]` where `gv_black = {title: ..., id: ...}` and so on. That structure will also pass the tests given the code, but if one didn't know the code... 
2- Is `send` used because the method is private or is it just a Ruby idiom? Testing the method call directly might be more readable. If private, then testing the public `unexportable?` might be an idea.

None of this blocking.

Reply:
I'll update the structure mentioned in point 1 since that seems more realistic to what we will see.

I'm using send because it is a private method. We could test `unexportable?`, but it was easier to wrap my head around testing two separate functions than one big function. When you start dropping a ton of `context` and `describe` blocks it gets very messy and hard to read. It's much cleaner the way I have it even if it's not the standard way of doing things.